sdk/communication/Azure.Communication.Chat/src/Models/ChatParticipant.cs

```diff
@@ -38,9 +43,14 @@ internal ChatParticipant(ChatParticipantInternal chatParticipantInternal)
         /// <summary> Time from which the chat history is shared with the member. The timestamp is in ISO8601 format: `yyyy-MM-ddTHH:mm:ssZ`. </summary>
         public DateTimeOffset? ShareHistoryTime { get; set; }
 
+        /// <summary>
+        /// Metadata
+        /// </summary>
+        public IDictionary<string, string> Metadata { get; }
```

Comment:
Since it doesn't have a setter, it should be set to an empty dictionary when ChatParticipant is constructed.

So that clients can do .Metadata.Add("key", "value")

I'm not sure but I think the way it is now .Metadata.Add() would throw a null reference exception, wouldn't it?

Reply:
public ChatParticipant(CommunicationIdentifier identifier)
        {
            User = identifier;
            Metadata = new ChangeTrackingDictionary<string, string>();
        }
        
        It is being set right in the constructor?